File: SoloLearn Codes/Youtube Link Finder/YoutubeLinkFinder.cs
using System;

namespace SoloLearn
{
    class Program
    {
        static void Main(string[] args)
        {
            string u = Console.ReadLine();
            Console.WriteLine(u.Contains("youtu.be") ? u.Substring("https://youtu.be/".Length) : u.Substring("https://www.youtube.com/watch?v=".Length));
        }
    }
}

File: SoloLearn Codes/Youtube Link Finder/YoutubeLinkFinder.java
import java.util.Scanner;
import java.util.regex.*;

public class YoutubeLinkFinder
{
    public static void main(String[] args) {
        String url = new Scanner(System.in).nextLine();
        System.out.println(url.contains("youtu.be") ? 
                url.substring("https://youtu.be/".length()) : 
                url.substring("https://www.youtube.com/watch?v=".length()));
    }
}